Client/Frontend/Browser/Tabs/RemoteTabsPanel.swift

```diff
@@ -576,6 +619,13 @@ class RemoteTabsTableViewController: UITableViewController {
         }
     }
 
+    func applyTheme() {
+        tableView.separatorColor = themeManager.currentTheme.colors.layerLightGrey30
+        if let delegate = tableViewDelegate as? RemoteTabsPanelErrorDataSource {
+            delegate.theme = themeManager.currentTheme
```

Comment:
Should this call `delegate.applyTheme(theme: themeManager.currentTheme)` instead? And then the theme variable could be private? If I get this right